newEvaluation.html

```diff
@@ -14,6 +14,7 @@
 	</nav>
 	
 	<!-- Section1 -->
+	<div id = "fieldsetContainer">
```

Comment:
This is more a matter of personal preference than anything else, as HTML is pretty permissive, but I personally feel that the extra spaces around the `=` makes the markup harder to read. Call me old-fashioned. :)